.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 8.9.4
+- 8.9.3
```

Comment:
Why the downgrade?

Reply:
As we discussed on HipChat, I think this should be the min version of `node` at `edX`.